symfony/client/src/orangehrmLeavePlugin/pages/LeaveAssign.vue

```diff
@@ -290,5 +372,29 @@ export default {
       return id && (id === 3 || id === 4);
     },
   },
+
+  watch: {
+    'leave.employee': function(employee) {
+      if (!employee) {
+        return;
+      } else {
+        this.http.request({
+             method: 'GET',
+             url: 'api/v2/pim/employees/' + this.leave.employee?.id + '/work-shift',
```

Comment:
try \`api/v2/pim/employees/${this.leave.employee.id}/work-shift\`

@super-chama @VageeshOrangeHRM  it's preferred to use `empNumber` over `id`, since `empNumber` is the primary key. e.g. /api/v2/pim/employees/{empNumber}/work-shift

Reply:
@RajithaKumara this employee object is coming from autocomplete-input which uses model object like `{ id : 1, label: ' ' }` since it's used everywhere.

Reply:
done